src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -186,9 +188,10 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     if (!poolAsset) return {}
     return {
       [poolAsset.chainId]: accountId,
-      [thorchainChainId]: runeAccountId ?? '',
+      // @TODO: Support multi accounts, but it's currently not supported anywhere in the Thorchain LP feature
```

Comment:
Not worse than before, but noting this still gives a false impression of type-safety, see develop vs. this diff:

<img width="979" alt="image" src="https://github.com/user-attachments/assets/413fc96b-dcc7-4c30-acf7-c2d043d30dfb">
<img width="965" alt="image" src="https://github.com/user-attachments/assets/30c3142a-4e95-4f13-a0ae-b0e9f161dc8b">

suggestion: If the rationale here is removing unsafe `T ?? ''` nullish coalescing, we may want to keep the current flavour, and keep things safe at type-level by updating `currentAccountIdByChainId` in `LpConfirmedWithdrawalQuote` to `Record<ChainId, undefined>`

Reply:
The rationale here is to use the first rune account id if `runeAccountId` is not defined, because `runeAccountId` is defined only if you are on the deposit side for now, the way it's designed is not intended to be set up if it's a withdraw, making every `Rune` sym and asym withdraws crashing

If the rune account id is not defined at the end, it will simply disable the button

The suggestion is then not what we want for sure

Reply:
Ohh very nice one indeed, `runeAccountId` is a state field and set only if you have a position, that makes more sense ty!

To disambiguate things here for anyone reading this, not only is it set if you have a position, but to be more accurate here, it is set if you have a position *with a `runeAddress` field*, i.e a sym/asym RUNE position.

Seems sane to me ser 🤝

Reply:
Not sure I understand why we need this still after reading. If it is a position that has rune meaning sym or asym rune, it will have the thorchain account id, otherwise it won't, but you wouln't be able to withdraw from those types anyways. I removed to test locally and am not seeing an issue (although I am seeing us incorrectly set the `pool.assetId` account to a thorchain account on asym rune positions)